packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.controller.js

```diff
@@ -129,7 +129,12 @@ export default class FlavorsListController {
   }
 
   isFlavorCategoryIncluded(category) {
+    const isUnavailableCategoryForTrustedZone = ['vps', 'iops'].includes(
```

Comment:
```suggestion
    const isCategoryUnavailableForTrustedZone = ['vps', 'iops'].includes(
```
don't you think it should come from featureFlipping?

Reply:
We have to use also feature flipping for categories.
My goals during this development is mainly to feature flippe menu products and configuration.

To feaure flippe categories, we have to create a dedicated task for each product and configuration menu.

Reply:
Finally i completed it.